### ISSUE TITLE
Unable to build new ego vehicle (gITF converter failed with exit code: 105)

### ISSUE THREAD

**Issue by FiIipe** [lgsvl/simulator#1935]
Hello, 

I've been trying to create a new ego vehicle to use with LGSVL. 
The vehicle that I'm trying to import is the Kia Niro and this is the .fbx file. 

The version of unity that I'm using is the 2020.3.3f1, the version of the simulator is the 2021.3 and the version of the Jaguar is also the 2021.3.

I've followed this tutorial and the video: https://www.svlsimulator.com/docs/simulation-content/add-new-ego-vehicle/

When I press build, I get the following errors: 

![image](https://user-images.githubusercontent.com/65399295/148063964-cb66cb65-8b4f-4ef9-8086-7132d37f879e.png)

Any help is very much appreciated. 

Cheers, 
Filipe

**Comment by EricBoiseLGSVL** [comment]
What OS and version are you running?

**Comment by FiIipe** [comment]
@EricBoiseLGSVL thank you for your quick response. I'm running on windows 10

**Comment by EricBoiseLGSVL** [comment]
hmm, we have not seen this issue internally but I have seen a few users with this issue. Let me look at the model

**Comment by EricBoiseLGSVL** [comment]
Did you make the fbx read write enabled?
![image](https://user-images.githubusercontent.com/45106230/148108913-c4167e86-44f6-41d2-b090-7f613ba97e69.png)

**Comment by FiIipe** [comment]
No, I did not change that field. Should I try again with read/write enabled?

**Comment by EricBoiseLGSVL** [comment]
yes please try that, then I have one more thing.  Be sure you export the material in the fbx importer.  I am building as well test.

**Comment by FiIipe** [comment]
Ok, I will try it. 
I'm not sure what you mean by this: "Be sure you export the material in the fbx importer". Can you explain, please?

**Comment by EricBoiseLGSVL** [comment]
I bet this is the issue.  I have successfully built the bundle locally.
The gltf exporter needs access to the materials to make changes to support the glb file.  The folders need to look like this:
![image](https://user-images.githubusercontent.com/45106230/148117507-dc57a3cc-87c0-46a4-bec6-aef65ffac9d0.png)

The export option is here in the fbx file like so:
![image](https://user-images.githubusercontent.com/45106230/148117650-e777616b-cda5-4d2b-9e1c-f9d551691443.png)

This will extract the material data to the folder you make in Models.  Then, the exporter has access to the materials.

**Comment by FiIipe** [comment]
@EricBoiseLGSVL The error still persists. I don't think the problem is the prefab or the fbx model. I tried to build the Jaguar model and I get exactly the same error. Can the problem be something wrong with the my unity installation or the simulator build? 

![image](https://user-images.githubusercontent.com/65399295/148202936-b5996ded-9333-4117-928c-71db1ac28e68.png)

If you managed to build the model, can you make it available on the LGSVL vehicle store? 

Cheers, 
Filipe

**Comment by EricBoiseLGSVL** [comment]
Can you show me the full path to the assetbundles folder?  The path seems strange.  Show the path to the assets you are building and the asset path to the AssetBundles folder like so:
![image](https://user-images.githubusercontent.com/45106230/148270066-31f0e32d-f172-48a1-abae-01eb2984dbf7.png)

and
![image](https://user-images.githubusercontent.com/45106230/148270155-0e1756da-3494-41dd-a8c7-6b0a9c3adcdb.png)

Sorry since I don't own the rights to the model I can't publish it.  You should also edit your previous post with the fbx

**Comment by FiIipe** [comment]
@EricBoiseLGSVL ok, I just removed the  .fbx model of the kia.

The path to the AssetBundles folder is the following: 

![image](https://user-images.githubusercontent.com/65399295/148297611-f24e3033-3ca1-475d-bffb-c095f04f6a0c.png)

Inside AssetBundles -> vehicles:

![image](https://user-images.githubusercontent.com/65399295/148297829-718149b0-f69d-4090-8dc4-e61482b48fbc.png)

The path to the fbx file: 

![image](https://user-images.githubusercontent.com/65399295/148297684-4087dc9e-73d4-4e54-adc1-b467b227e259.png)

Is there anyway that you can send me the file instead of publishing it?

Cheers, Filipe

**Comment by FiIipe** [comment]
@EricBoiseLGSVL, I was able to build it, finally!

I was trying to build the vehicle in the following directory: `C:\Users\Filipe Cerveira\Desktop\simulator`. 
I moved the entire folder from that directory to the root of disk C, `C:\simulator`, and I tried to build it again. Everything worked smoothly, no problems. It seems like the space between Filipe Cerveira was messing up the build process for some unknown reason. I will try to build it again in a directory with space to make sure that the problem was the space.
